tests/LinAlg/matrixTests.hpp

```diff
@@ -1113,37 +1214,7 @@ class MatrixTests : public TestBase
       hiop::hiopMatrixDense* A,
       const local_ordinal_type row,
       const real_type val) = 0;
-  virtual real_type getLocalElement(
-      const hiop::hiopMatrixDense* a,
-      local_ordinal_type i,
-      local_ordinal_type j) = 0;
-  virtual real_type getLocalElement(
-      const hiop::hiopVector* x,
-      local_ordinal_type i) = 0;
-  virtual local_ordinal_type getNumLocRows(hiop::hiopMatrixDense* a) = 0;
```

Comment:
We need to keep methods that return local numbers of rows and columns as unit test helpers, because it is not certain all matrix implementations will have such public methods. 

Furthermore, this change introduces a bug as `m()` and `n()` are **not** local matrix sizes.

Reply:
Sounds good. I will revert this change and keep the helper methods